src/main/java/net/minecraftforge/event/entity/living/LivingEnchantmentLevelEvent.java

```diff
@@ -0,0 +1,45 @@
+package net.minecraftforge.event.entity.living;
+
+import net.minecraft.enchantment.Enchantment;
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.item.ItemStack;
+
+/**
+ * This event is fired when an enchantment level is being looked up for an entity.
+ * @author rubensworks
```

Comment:
The class you are referring has a lot of obsolete information IMO, I don't think it's a good idea to have documentation just for the documentation. The current javadoc here should tell everything that is required.

I didn't know the author tag is not supposed to be used, since I've seen it a couple of times in the Forge source code. Maybe someone else can clarify this?